docs/examples/src/bin/dynamic_components.rs

```diff
@@ -0,0 +1,320 @@
+#![allow(clippy::unused_unit)]
+
+use kompact::prelude::*;
+use std::sync::Arc;
+use kompact::component::AbstractComponent;
+use std::fmt;
+use std::io::{stdin, BufRead};
+use std::error::Error;
+
+// ANCHOR: simple_components
+// like ignore_lifecycle!, but a tad louder
+macro_rules! info_lifecycle {
```

Comment:
Move this macro into core, please. I think this could be quite useful for other people as well, actually.